pkg/rpc/server/server_test.go

```diff
@@ -620,6 +620,54 @@ var rpcTestCases = map[string][]rpcTestCase{
 				assert.NotEqual(t, 0, res.GasConsumed)
 			},
 		},
+		{
+			name:   "positive, good witness",
+			params: `["5707000c14010c030e05060c0d020e0f0d030e070900000000db307068115541f827ec8c21aa270700000011400c140d0f03020900020103070304050201000e060c09db307169115541f827ec8c21aa270700000012401340",["0x0000000009070e030d0f0e020d0c06050e030c01","0x090c060e00010205040307030102000902030f0d"]]`,
```

Comment:
This contains script of the `testverify.avm` in hex, am I right?

Reply:
Yes, I'll add a comment.